iree/tools/iree-e2e-matmul-test.c

```diff
@@ -401,19 +435,19 @@ static iree_status_t iree_check_matmul_failure(
 
   fprintf(file, "\n");
   print_matrix(file, "left-hand side", PRECISION_LOW, m_start, m_end, k_start,
-               k_end, lhs, NULL);
+               k_end, lhs, NULL, NULL);
   fprintf(file, "\n");
   print_matrix(file, "right-hand side", PRECISION_LOW, k_start, k_end, n_start,
-               n_end, rhs, NULL);
+               n_end, rhs, NULL, NULL);
   fprintf(file, "\n");
   print_matrix(file, "input accumulator", PRECISION_LOW, m_start, m_end,
-               n_start, n_end, acc, NULL);
+               n_start, n_end, acc, NULL, NULL);
   fprintf(file, "\n");
   print_matrix(file, "expected result", PRECISION_LOW, m_start, m_end, n_start,
-               n_end, expected_result, actual_result);
+               n_end, expected_result, actual_result, "🦄");
   fprintf(file, "\n");
   print_matrix(file, "actual result", PRECISION_LOW, m_start, m_end, n_start,
-               n_end, actual_result, expected_result);
+               n_end, actual_result, expected_result, "🐛");
```

Comment:
Personally I found the :poop: emoji pretty clear. "bug" is clever but I think actually harder to read. So for more than just puerile reasons I would leave it as :poop:. Don't have strong opinions on the emoji to use for the corresponding expected result, since that's really just highlighting the value to compare it to